content/lessons/chapter-1/transacting-1.tsx

```diff
@@ -1,5 +1,3 @@
-'use client'
-
 import { useTranslations } from 'hooks'
```

Comment:
```suggestion
'use client'

import { useTranslations } from 'hooks'
```

Reply:
Addressed in 895cf4ec38730c26699aa697bed6182c73d66f51